web/packages/teleport/src/Bots/Add/GitHubActions/AddBotToWorkflow.tsx

```diff
@@ -0,0 +1,142 @@
+import Flex from 'design/Flex';
+
+import useTeleport from 'teleport/useTeleport';
+
+import { FlowButtons } from '../Shared/FlowButtons';
+
+import { FlowStepProps } from '../Shared/GuidedFlow';
+
+import { useGitHubFlow } from './useGitHubFlow';
+
+export function AddBotToWorkflow({ prevStep, nextStep }: FlowStepProps) {
+  const { tokenName, createBotRequest } = useGitHubFlow();
+  const ctx = useTeleport();
+  const cluster = ctx.storeUser.state.cluster;
+
+  const yaml = getWorkflowExampleYaml(
+    createBotRequest.botName,
+    cluster.authVersion,
+    cluster.publicURL,
+    tokenName
+  );
+
+  return (
+    <Box mb="0">
+      <Text bold fontSize={4} mb="3">
+        Step 3: Connect Your Bot in a GitHub Actions Workflow
+      </Text>
+      <Text fontSize={3} mb="3">
+        Teleport has created a role, a bot, and a join token. Below is an
+        example GitHub Actions workflow doc to help you get started. You can
+        find this again from the bot’s options dropdown
+      </Text>
+      <Flex
+        flex="1"
+        height="630px"
+        maxWidth="840px"
+        mb="3"
+        pt="3"
+        pr="3"
+        bg="levels.deep"
+        borderRadius={3}
+      >
+        <TextEditor
+          readOnly={true}
+          bg="levels.deep"
+          data={[{ content: yaml, type: 'yaml' }]}
+          copyButton={true}
+          downloadButton={true}
+          downloadFileName={`${createBotRequest.botName}-githubactions.yaml`}
+        />
+      </Flex>
+      <FlowButtons
+        nextStep={nextStep}
+        prevStep={prevStep}
+        isLast={true}
+        backButton={{
+          hidden: true,
+        }}
+      />
+    </Box>
+  );
+}
+
+function getWorkflowExampleYaml(
+  botName: string,
+  version: string,
+  proxyAddr: string,
+  tokenName: string
+): string {
+  return `on:
+push:
+  branches:
+  - main
+jobs:
+demo:
+  permissions:
+    # The "id-token: write" permission is required or Machine ID will not be
+    # able to authenticate with the cluster.
+    id-token: write
+    contents: read
+  # if you added a workflow name in the previous step, make sure you use the same value here
+  name: ${botName}-example
+  runs-on: ubuntu-latest
+  steps:
+  - name: Checkout repository
+    uses: actions/checkout@v3
+  - name: Fetch Teleport binaries
+    uses: teleport-actions/setup@v1
+    with:
+      version: ${version}
+  # server access example
+  - name: Fetch credentials using Machine ID
+    id: auth
+    uses: teleport-actions/auth@v2
+    with:
+      proxy: ${proxyAddr}
+      token: ${tokenName}
+      # Enable the submission of anonymous usage telemetry. This
+      # helps us shape the future development of \`tbot\`. You can disable this
+      # by omitting this.
+      anonymous-telemetry: 1
+  - name: List nodes (tsh)
+    # Enters a command from the cluster, in this case "tsh ls" using Machine
+    # ID credentials to list remote SSH nodes.
+    run: tsh ls
+  # kubernetes access example
```

Comment:
We should remove the k8s example as we don't setup a role which grants access to Kubernetes resources.